### ISSUE TITLE
Training configuration (number of GPUS and epoch durations)

### ISSUE THREAD

**Issue by basma-b** [baidu/Dialogue#14]
Can you please provide information about the hardware configuration of the machine on which you trained your model (GPU, RAM, Disk ..) ? 
the duration of one epoch and how for how many epochs did you train your model ?

Thanks

**Comment by xyzhou-puck** [comment]
Hi，

GPU：Nvidia Tesla P40
RAM: 252G
# CPU: 34
DISK: 7T

Xiangyang

**Comment by xyzhou-puck** [comment]
@luluxing3 .

**Comment by luluxing3** [comment]
It takes about 8 hours of one epoch without parallelism on Nvidia Tesla P40.

**Comment by basma-b** [comment]
Thanks, and you obtain the reported results on your paper after how many epochs ? @luluxing3

**Comment by luluxing3** [comment]
We pass 3 epochs and choose the best model base on validation set, and we get the best model about the end of one epoch.
